### PR TITLE
Anbox docs test and other fixes

### LINKED ISSUE #104: Setup views to display the discourse documentation

### PR DESCRIPTION
**Pull request by carkod**
## Done

- Fix topic id (it wasn't showing navigation)
- Add test
- Project clean up

## QA

- Check out this feature branch
- Run the site using the command `./run serve` or `dotrun`
- View the site locally in your web browser at: http://0.0.0.0:8043/docs
- Make sure it shows the navigation and it can be toggled (it should show more than 1 item)
- Make sure global nav is not missing
- Make sure tests pass the checks

## Issue / Card

Fixes #104

### CODE REVIEW

File: templates/docs/document.html [hunk 1]
```diff
@@ -30,11 +30,11 @@ <h1>{{ document.title }}</h1>
       {{ document.body_html | safe }}
       <hr />
       <p><i>Last updated {{ document.updated }}.</i></p>
-      <div class="p-notification--information">
+      <!-- <div class="p-notification--information">
```
Comment (carkod) [inline templates/docs/document.html:33]: They don't want it to be collaborative yet, but in the future we will add this.

File: webapp/app.py [hunk 2]
```diff
@@ -44,8 +44,7 @@
         api=DiscourseAPI(
             base_url="https://discourse.ubuntu.com/", session=session
         ),
-        category_id=49,
```
Comment (carkod) [inline webapp/app.py:47]: Optional category id (v1.0.0)

File: package.json [hunk 3]
```diff
@@ -2,9 +2,10 @@
   "author": "Canonical webteam",
   "license": "LGPL v3",
   "scripts": {
-    "start": "yarn run build && yarn run serve",
+    "start": "yarn run build && concurrently --kill-others --raw 'yarn run watch-css' 'yarn run watch-js' 'yarn run serve'",
     "clean": "rm -rf node_modules yarn-error.log css static/css *.log *.sqlite _site/ build/ .jekyll-metadata .bundle",
-    "watch": "watch -p 'static/sass/**/*.scss' -c 'yarn run build'",
+    "watch-css": "watch -p 'static/sass/**/*.scss' -p 'node_modules/vanilla-framework/scss/**/*.scss' -c 'yarn run build-css'",
```
Comment (anthonydillon) [inline package.json:7]: Why are you watching for changes to vanilla-framework in node_modules?

File: package.json [hunk 4]
```diff
@@ -2,9 +2,10 @@
...
+    "watch-js": "watch -p 'static/js/src/**/*.js' -p 'static/js/data/**/*.js' -p 'static/js/third-party/**/*.js' -c 'yarn run build-js'",
```
Comment (anthonydillon) [inline package.json:8]: Should we watch third-party?

File: templates/docs/document.html (hunk 1 above)
Comment (anthonydillon) [reply templates/docs/document.html:33]: Can you remove the notification until ready?

**Comment by anthonydillon** [comment]
Can you add `u-no-padding` class to the div with the `row` class in the docs template, please.

**Comment by carkod** [comment]
@anthonydillon done. Is the QA failing? Can you give me some details of where it is failing?

**Comment by carkod** [comment]
All right. Thanks

**Pull request merged**